pkg/windows/windows.go

```diff
@@ -591,11 +591,36 @@ func (vm *windows) createDirectories() error {
 // removeDirectories removes all directories created as part of the configuration process
 func (vm *windows) removeDirectories() error {
 	vm.log.Info("removing directories")
+	existingFiles, err := getFilesToTransfer()
+	if err != nil {
+		return errors.Wrapf(err, "error getting list of files transferred to the instance")
+	}
+	for src, dest := range existingFiles {
+		filePath := dest + "\\" + filepath.Base(src.Path)
+		err = wait.Poll(retry.Interval, retry.Timeout, func() (bool, error) {
```

Comment:
Why does this polling need to be introduced?
If an error occurs the instance deconfigure wll be re-queued by the controller-runtime manager.
The node object will still exist at this point, so that won't be a problem.

Reply:
I had suggested this in an earlier [comment](https://github.com/openshift/windows-machine-config-operator/pull/1482#discussion_r1140443886) but I am not set on it. I understand you are suggesting the regular k8s pattern but we need to be pragmatic. These reconcile roundtrips are becoming expensive because of SSH failures. But I am not set on it and can be convinced otherwise.

Reply:
If we are worried about time here, I think the best thing to do would be to condense this into as little powershell commands as possible, `rmDirCmd()` returns the PS equivalent of `rm -rd -f`.

Depending on the outcome of [this disscusion](https://github.com/openshift/windows-machine-config-operator/pull/1482#discussion_r1142802352) this could be simplified by deleting `remoteDir` (C:\Temp) and `k8sdir` (C:\k). All the other files will be removed as part of that.


As a side note, if polling needs to be used, `PollImmediate` should be used instead of `Poll`, the code as is introduces `len(existingFiles) * retry.Interval` unnecessary waiting time.